docs/infrastructure/administration.md

```diff
@@ -0,0 +1,416 @@
+# Infrastructure administration
+
+## Obtaining access
+
+- Heroku: Follow the [Mozilla Heroku SSO guide] then ask to be invited to the Heroku apps.
```

Comment:
To give some context - the target audience I had in mind when writing the "Obtaining access" section, was someone joining the Treeherder team, or someone from another team who needs access for eg a Treeherder feature they are adding. As such, the objectives were:
* to ensure the person even knows what access they need to be asking for (since without knowing what systems/tools are used it's hard to know what one is even missing)
* to give a hint to the Treeherder team members as to how to give them that access (since they might forget what's controlled where, if it's been a while)

I had initial wordings of this section that were more verbose, but in the end they didn't really add much more to meeting those objectives than the current more concise version. (That and the text wrapping looked awful in a couple of cases.)

> I have access to Heroku, however, I can't find `treeherder` under [mozillacorporation](https://dashboard.heroku.com/teams/mozillacorporation/apps) when I search for it.
> There might be a bug on the Heroku list of apps. Doing an in-page search for "treeherder" shows the pipeline.

A don't believe this is a bug - the three Treeherder Heroku apps all belong to the Pipeline `treeherder` (for the app names, see the docs section below this one), so are grouped together under it. Click the expandy-icon next to the pipeline to see the apps.

> As of now, this seems to be the instructions on how to get Heroku access for the first time rather than for someone who already is a Heroku member.

I don't follow. You can't give yourself access to a Heroku app, someone on the Treeherder team will need to do so. However before they can, you need to have a Heroku account set up via SSO. Hence why this section says to follow the SSO guide then "ask to be invited to the Heroku apps".